trestle/tasks/cis_to_component_definition.py

```diff
@@ -475,6 +479,29 @@ def _build_control_implementation(
             )
         return control_implementation
 
+    def _get_title(self, atom, root):
+        """Extract rule title from compliance-as-code rule.yml."""
+        title = None
+        for path, dirs, _files in os.walk(root):
+            if atom in dirs:
+                folder = os.path.join(path, atom)
+                tpath = pathlib.Path(folder) / 'rule.yml'
+                fp = pathlib.Path(tpath)
+                f = fp.open('r', encoding=const.FILE_ENCODING)
+                content = f.readlines()
+                f.close()
+                for line in content:
+                    if line.startswith('title:'):
+                        title = line.split('title:')[1]
+                        break
+        if title is None:
+            msg = f'unable to find "{atom}"'
+            logger.error(msg)
+            raise RuntimeError(msg)
+        title = title.strip().strip("'").strip('"')
```

Comment:
Can order of striped characters change or it is always like that?

Reply:
The title is surrounded by either single quotes or double quotes, but not both. So the order for stripping does not matter.